macrofusion.py

```diff
@@ -390,7 +397,28 @@ def __init__(self):
         self.check_desatmeth = self.gui.get_object("check_desatmeth")
         self.combobox_desatmet = self.gui.get_object("combobox_desatmet")
         self.combobox_desatmet.set_active(settings["fuse_settings"]["gray-projector"][1])
- 
+        
+        self.check_corrthres = self.gui.get_object("check_corrthres")
+        self.spinbuttoncorrthres = self.gui.get_object("spinbuttoncorrthres")
+        self.ajus_corrthres = Gtk.Adjustment(value=0.9, lower=0, upper=1, step_increment=0.1, page_increment=0.1, page_size=0)
+        self.spinbuttoncorrthres.set_adjustment(self.ajus_corrthres)
+        self.spinbuttoncorrthres.set_digits(2)
+        self.spinbuttoncorrthres.set_value(settings["align_settings"]["corr_thres"][1])
+
+        self.check_pnt = self.gui.get_object("check_pnt")
+        self.spinbuttonpnt = self.gui.get_object("spinbuttonpnt")
+        #self.ajus_corrthres = Gtk.Adjustment(value=0, lower=0, upper=1, step_increment=0.1, page_increment=0.1, page_size=0)
+        #self.spinbuttoncorrthres.set_adjustment(self.ajus_corrthres)
+        #self.spinbuttoncorrthres.set_digits(1)
```

Comment:
Would you mind removing all the commented code? Looks like it is not going to be very useful in the future